demos/jans-tent/CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing to Auth-tdd-client
```

Comment:
you can take this file out. Just point to the jans contributing guide

Reply:
done